README.md

```diff
@@ -123,7 +124,7 @@ for more information about this package.
 
 > **Note:**  
 > Google Cloud Functions support for events predates the CNCF Cloud
-> Events initiative. The types in the `Google.Cloud.Functions.Framework.GcfEvents`
+> Events initiative. The types in the `OpenFunction.Framework.GcfEvents`

```

Comment:
Thanks very much for the PR!

GcfEvents means Google cloud function events, maybe we can change it to OpenFunctionEvents in another PR.